modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,80 @@
+link:https://issues.redhat.com/browse/GITOPS-3947[GITOPS-3947]
+
+[id="fixed-issues-1-12-2_{context}"]
+== Fixed issues
+
+* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE, and EKS clusters as this binary was not available in the GitOps container. This update fixes the issue as the `argocd-k8s-auth` binary has been added in the GitOps container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
+
+* Before this update, when Argo CD attempted to connect to Azure DevOps, an error was displayed due to the deprecation of the `rsa-ssh` host key algorithm, by the Azure DevOps Repository service. This update fixes the issue as Argo CD provides support for the `rsa-ssh` host key algorithms during the communication process between Argo CD and Azure DevOps Repository service. link:https://issues.redhat.com/browse/GITOPS-4543[GITOPS-4543]
+
+* Before this update, {gitops-title} did not schedule the workloads on infrastructure nodes when the `runOnInfra` field was enabled in the `GitOpsService` custom resource (CR). This update fixes the issue by introducing a `node-selector` for infrastructure nodes on the {gitops-title} plugin workloads, enabling users to configure `Tolerations` on the `gitops-console` pod. As a result, when the `runOnInfra` field is enabled, the `gitops-console` pod is placed on the infrastructure nodes similar to the default workloads. link:https://issues.redhat.com/browse/GITOPS-4496[GITOPS-4496]
+
+* Before this update, the `ignoreDifferences` sync option in Argo CD did not work for array fields. This update fixes the issue by modifying the merge strategy of the `ignoreDifferences` sync option used in the upstream project to handle array fields. As a result, the sync option now functions correctly by allowing users to ignore specific elements in the array during sync. link:https://issues.redhat.com/browse/GITOPS-2962[GITOPS-2962]
+
+* Before this update, users were unable to include context for notifications, in the `NotificationsConfiguration` custom resource (CR). With this update, users can now add notifications by utilizing the `context` field within the `NotificationsConfiguration` (CR). This field enabled users to add context in the `NotificationsConfiguration` (CR) and can also be utilized to establish shared context across all templates defined in the `NotificationsConfiguration` (CR). link:https://issues.redhat.com/browse/GITOPS-4304[GITOPS-4304]
+
++
+Example for `context`
++
+[source,yaml]
+----
+spec:
+  context:
+    key1: value1
+    key2: value2
+  templates:
+    template.app-created: |-
+      email:
+        subject: Application {{.app.metadata.name}} has been created.
+      message: Application {{.app.metadata.name}} has been created.
+      teams:
+        title: Application {{.app.metadata.name}} has been created.
+    template.app-deleted: |-
+     ...
+     ...
+----
```

Comment:
Because we only want to showcase context example, shouldn't we omit the templates part?

Also, these are variables, if I understand it correctly. It should be more like this:
```suggestion
spec:
  context:
    <key>: <value> #<1>
----
<1> Replace ` <key>` with <add explanation here> and `<value>` with <add explanation here> . You can add as many key-value pairs as needed.
```
I would check for explanation with SME.

Reply:
Fixed.